Doc/whatsnew/3.9.rst

```diff
@@ -262,6 +262,8 @@ Changes in the Python API
 * The :mod:`venv` activation scripts no longer special-case when
   ``__VENV_PROMPT__`` is set to ``""``.
 
+* The :mod:`importlib` module now ignore the :envvar:`PYTHONCASEOK`
+  environment variable when :option:`-E` or :option:`-I` command line option is used.
```

Comment:
Suggestion for better English:

"... when the :option:\`-E\` or the :option:\`-I\` command line options are used.